docs/integrations/positioning-systems/ublox-evk-m8n.mdx

```diff
@@ -0,0 +1,267 @@
+u-blox 7, u-blox 8 and u-blox M8 positioning products simple. The built-in USB
+interface provides both power supply and high-speed data transfer, and
+eliminates the need for an external power supply. The u-blox evaluation kits are
+compact, and their user-friendly interface and power supply make them ideally
+suited for use in laboratories, vehicles and outdoor locations.
+
+In this tutorial we will present the process of connecting a UBlox GPS module to
+your Leo Rover and integrating it with the system on the **UBlox EVK-M8N GPS
+module** example.
+
+The steps might slightly differ for other UBlox models, but should be
+essentially similar.
+
+## What to expect?
+
+After completing this tutorial, you should be able to gather the GPS data and
+get it published on ROS topics.
+
+<ImageZoom
+  loading="eager"
+  src="/img/robots/leo/integrations/ublox-gps/ech-ublox-gps-node.webp"
+  width="294"
+  height="431"
+  figStyle={{
+    width: 400,
+  }}
+/>
+
+## Prerequisites
+
+<LinkButton
+  to="/leo-rover/1.8/advanced-guides/ros-development"
+  title="ROS Development"
+  description="Detailed guide on ROS development for Leo Rover, covering topics like adding
+  additional functionalities, building ROS packages and more."
+/>
+
+## Referenced products
+
+<ProductPreview
+  shopUrl="https://www.leorover.tech/shop/gps-module"
+  imageSrc="/img/robots/leo/integrations/ublox-gps/ublox-evk-m8n-shop.webp"
+  width={1200}
+  height={802}
+  title="UBlox EVK-M8N GPS module"
+  description="Enhance Leo Rover's capabilities with the UBlox EVK-M8N GPS module.
+    Experience precise navigation and versatile connectivity with multi-GNSS support,
+    ensuring high-performance and reliability."
+/>
+
+## List of components
+
+- UBlox GPS module (we have used the **UBlox EVK-M8N GPS module**)
+- Any computer which you can connect to the rover via ssh.
+
+## Mechanical integration
+
+When it comes to mounting the module, any solution that works for you is okay.
+You can 3D print a special mounting plate, use a velcro or even a tape.
+
+## Wiring and electronics connection
+
+Wiring up the module is very simple, as it doesn't require external power
+supply. You just need to connect the module with a USB cable with the rover via
+the USB socket in the mounting plate. There is a micro-usb mounting hole with
+the USB sign on the module, and you need to plug in the USB cable there.
+
+## Software integration
+
+The first thing you can do is to make sure your device has the correct
+permissions and is available at the fixed path on your system. To do this, you
+can add the following rule to the `udev` service. We will need `idVendor` and
+`idProduct`, to make the rules for the correct device. To get that information,
+you need to connect to the rover via `ssh`, and then, you can use the `lsusb`
+command. It will list all connected usb devices. In the output, you should look
+for something named like **U-Blox AG [u-blox 8]**:
```

Comment:
This is quite a long paragraph
```suggestion
The first thing you can do is to make sure your device has the correct
permissions and is available at the fixed path on your system.

We will need USB Vendor and Product IDs to make the rules for the correct device. To get that information,
you need to connect to the rover via `ssh`, and then, you can use the `lsusb`
command. It will list all connected usb devices. In the output, you should look
for something named like **U-Blox AG [u-blox 8]**:
```

Reply:
Done